config.lua

```diff
@@ -129,3 +129,55 @@ Config.CharacterDataTables = {
     ['player_outfits'] = 'citizenid',
     ['player_vehicles'] = 'citizenid',
 } -- Rows to be deleted when the character is deleted
+
+---@param sexString number | string 
+---@return string
+local function getSexString(sexString)
+    if sexString ~= 1 then
+        sexString = 'M'
+    else
+        sexString = 'F'
+    end
+    return sexString
+end
+
+---@param item string
+---@param itemType string
+---@return table
+local function getItemMetadata(item, itemType)
+    local metadata
+    if type(item) ~= "string" or type(itemType) ~= "string" then return end
+
+    if itemType == 'id' then
+        metadata = {
+            type = string.format('%s %s', PlayerData.charinfo.firstname, PlayerData.charinfo.lastname),
+            description = string.format('CID: %s  \nBirth date: %s  \nSex: %s  \nNationality: %s', PlayerData.citizenid, PlayerData.charinfo.birthdate, getSexString(PlayerData.charinfo.gender), PlayerData.charinfo.nationality)
+        }
+    else
+        metadata = {
+            type = 'License',
+            description = string.format('First name: %s  \nLast name: %s  \nBirth date: %s', PlayerData.charinfo.firstname, PlayerData.charinfo.lastname, PlayerData.charinfo.birthdate)
+        }
+    end
+    metadata = {
+        cardtype = item,
+        citizenid = PlayerData.citizenid,
+        firstname = PlayerData.charinfo.firstname,
+        lastname = PlayerData.charinfo.lastname,
+        birthdate = PlayerData.charinfo.birthdate,
+        sex = getSexString(PlayerData.charinfo.gender),
+        nationality = PlayerData.charinfo.nationality,
+        mugShot = 'none',
+    }
+    return metadata
+end
+
+---@alias ItemName string
+---@alias ItemAmount number
+---@alias ItemMetadata table
+---@type table<ItemName, ItemAmount, ItemMetadata>
```

Comment:
```suggestion
---@type { item: string, amount: integer, metadata: table|function }
```
I don't believe you can specify a table<> with more than two values